tests/emukit/test_acquisitions.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import pytest
+import pytest_lazyfixture
+from scipy.optimize import check_grad
+
+from bayesian_optimization.test_entropy_search import entropy_search_acquisition
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement, NegativeLowerConfidenceBound
+from emukit.bayesian_optimization.acquisitions.entropy_search import MultiInformationSourceEntropySearch
+from emukit.bayesian_optimization.acquisitions.log_acquisition import LogAcquisition
+from emukit.core import ParameterSpace, ContinuousParameter, InformationSourceParameter
+from emukit.core.acquisition.acquisition_per_cost import CostAcquisition
+
+from emukit.bayesian_optimization.acquisitions import ProbabilityOfImprovement
+from emukit.experimental_design.model_based.acquisitions import ModelVariance, IntegratedVarianceReduction
+from emukit.quadrature.acquisitions import SquaredCorrelation
+
+default_grad_tol = 1e-7
+gradient_acquisitions = [('negative_lower_confidence_bound_acquisition', default_grad_tol),
```

Comment:
Is there any way these could be named tuples, dictionaries, or anything like that? If we add say one or two more parameters here, it would be next to impossible to guess what each number exactly means. Even 1e-3 does not tell you much (must be a precision? of what?)

Reply:
Done